File: pset6/mario.py
'''
builds mario pyramid
with user entering desired height during runtime
like in pset1
Usage: python mario.py
'''

import cs50

while True:
    print("Height: ", end = "")
    height = cs50.get_int()
    if (height > 0) and (height < 24):
        break
    
#for each row
for row in range(height):
    
    #print spaces
    for j in range(height - (row+1)):
        print(" ", end = "")
    
    #print hashes
    for k in range(row+2):
        print("#", end = "")
    
    print("")
    



        

File: pset1/mario.c
/*
User enters height of mario pyramid during
runtime and program prints it of that size.

Usage: mario
*/

#include <stdio.h>
#include <cs50.h>

int main(void)
{
    //ask user
    
    int height;
    do
    {
        printf("Height: ");
        height = GetInt();
        
        if (height == 0)
            return 0;
            
        if (height > 23)
            printf("Height must be between 1 and 23\n");
            
    }
    while(height < 1 || height > 23);
    
    
    //print pyramid
    
    int row;
    
    for (row = 1; row <= height; row++)
    {
        for (int i = 0; i <= (height-row); i++)
        {
            printf(" ");
        }
        for (int i = 0; i <= row; i++)
        {
            printf("#");
        }
    
        printf("\n");
    }
    
}

